File: expectanim/src/main/java/com/github/florent37/expectanim/core/alpha/AlphaAnimExpectation.java
package com.github.florent37.expectanim.core.alpha;

import android.view.View;

import com.github.florent37.expectanim.core.AnimExpectation;

/**
 * Created by florentchampigny on 17/02/2017.
 */

public abstract class AlphaAnimExpectation extends AnimExpectation{
    public abstract Float getCalculatedAlpha(View viewToMove);
}

File: expectanim/src/main/java/com/github/florent37/expectanim/core/alpha/AlphaAnimExpectation.kt
package com.github.florent37.expectanim.core.alpha

import android.view.View

import com.github.florent37.expectanim.core.AnimExpectation

abstract class AlphaAnimExpectation : AnimExpectation() {
    abstract fun getCalculatedAlpha(viewToMove: View): Float?
}
